### DISCUSSION TITLE
Follow redirects

### DISCUSSION THREAD

**Discussion by ri4a** [phax/phoss-smp#241] [Q&A]
Hi,

Will the phossSMP server REST API follow SMP redirects?

What about https://mvnrepository.com/artifact/com.helger.peppol/peppol-smp-client ?

Thanks!

**Comment by phax** [comment]
Hi @ri4a, thanks for asking.
Indeed the SMP Clients (for Peppol, OASIS BDXR v1 and OASIS BDXR v2) do support the handing of SMP redirects by default.
Via the method `setFollowSMPRedirects(boolean)` that can also be disabled on the fly if theoretically needed.

hth, Philip
